frontend/components/JoinGraveBtn.tsx

```diff
@@ -648,16 +673,59 @@ export default function JoinGraveBtn({
       );
     } else {
       return (
-        <Button
-          onClick={handleJoin}
-          disabled={loading}
-          mb="10px"
-          fontFamily="Bungee"
-          fontSize="16px"
-          fontWeight="400"
-        >
-          {loading ? 'Processing...' : 'START'}
-        </Button>
+        <>
+          <AlertDialog
+            motionPreset="slideInBottom"
+            leastDestructiveRef={cancelRef}
+            onClose={() => {
+              setLoading(false);
+              onClose();
+            }}
+            isOpen={isOpen}
+            isCentered
+          >
+            <AlertDialogOverlay />
+            <AlertDialogContent>
+              <AlertDialogHeader>Found existing URDs</AlertDialogHeader>
+              <AlertDialogCloseButton />
+              <AlertDialogBody>
+                You currently have a Universal Receiver Delegate set for LSP7 (
```

Comment:
what about something like this:
```
"Hey, just a heads up! You've got a Universal Receiver Delegate set up for LSP7 ({URDLsp7}) and maybe for LSP8 ({URDLsp8}) too. If you jump into the Grave, it's gonna switch up your current URD. This could mess with some stuff on your UP profile, especially the parts that rely on these."
```